### PR TITLE
Translate triggering-a-deployment.md into Japanese

### PR DESCRIPTION
**Pull request by ono-max**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/ja/docs/user-guide/triggering-a-deployment.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+PipeCD は Git をシングルソースオブトゥルース（全ての Application リソースが Git で宣言的かつ不変で定義されている）として扱っています。
+開発者は Application やインフラに変更を加える際は Git のリポジトリにプルリクエストを作成する必要があります。
+クラスター内で実行している Application やインフラにとって Git 内で定義されている状態は望ましい状態になります。
+
+PipeCD は Deployment を実行することで、クラスターで実行しているリソースに Git で加えられた変更を適用します。 
+Deployment のミッションはクラスター内の Application の全てのリソースと Git の最新のコミットの状態の差分をなくすことです。
+
+デフォルトでは新しくマージされたプルリクエストが Application に変更を加える際に、クラスター内の Application と同期を取る為に新しい Deployment がトリガーされます。
+新しい Deployment のトリガーのタイミングについて設定することも可能です。
+例えば [`onOutOfSync`](/docs/user-guide/configuration-reference/#deploymenttrigger) を使うことで、ドリフト検出がされると `OUT_OF_SYNC` State を解決するために Deployment がトリガーされるようになります。
```
Comment (ono-max) [inline docs/content/ja/docs/user-guide/triggering-a-deployment.md:18]: I'm not sure if this part of the translation is correct.

**Comment by ono-max** [comment]
@nakabonne @knanao 

Could you review this PR?
Thx.

File: docs/content/ja/docs/user-guide/triggering-a-deployment.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Deployment のトリガー"
+linkTitle: "Deployment のトリガー"
+weight: 2
+description: >
+  Deployment の自動トリガーや手動でのトリガーについて
+---
+
+PipeCD は Git をシングルソースオブトゥルース（全ての Application リソースが Git で宣言的かつ不変で定義されている）として扱っています。
```
Comment (nakabonne) [inline docs/content/ja/docs/user-guide/triggering-a-deployment.md:9]: In a Japanese post, it tends to be this.
https://ja.wikipedia.org/wiki/%E4%BF%A1%E9%A0%BC%E3%81%A7%E3%81%8D%E3%82%8B%E5%94%AF%E4%B8%80%E3%81%AE%E6%83%85%E5%A0%B1%E6%BA%90

```suggestion
PipeCD は Git を信頼できる唯一の情報源（全ての Application リソースが Git で宣言的かつ不変で定義されている）として扱っています。
```

File: docs/content/ja/docs/user-guide/triggering-a-deployment.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Deployment は定められた Pipeline として実行され、それぞれの Stage のログなどを含んだ実行については Deployment 詳細ページにてリアルタイムで確認することができます。
+
+![](/images/deployment-details.png)
+<p style="text-align: center;">
+A Running Deployment at the Deployment Details Page
+Deployment 詳細ページ例
+</p>
+
+先述したようにデフォルトでは全ての Deployment はマージされたコミットを確認することで自動でトリガーされますが、Web コンソールから手動で新しい Deployment をトリガーすることもできます。
+Application 詳細ページの `SYNC` ボタンをクリックすると、 master （デフォルト）ブランチの最新のコミットの状態と差分をなくす為に新しい Deployment をトリガーします。
```
Comment (nakabonne) [inline docs/content/ja/docs/user-guide/triggering-a-deployment.md:38]: Truthfully all places where is supposed to be master should get revised.
```suggestion
Application 詳細ページの `SYNC` ボタンをクリックすると、 デフォルトブランチの最新のコミットの状態と差分をなくす為に新しい Deployment をトリガーします。
```

**Comment by nakabonne** [comment]
Way to go!
/approve

**Pull request merged**